packages/transformer/src/IModelTransformer.ts

```diff
@@ -2494,7 +2496,10 @@ export class IModelTransformer extends IModelExportHandler {
                   ElementRefersToElements.classFullName,
                   targetRelInstanceId
                 );
-              this.importer.deleteRelationship(targetRelationship.toJSON());
+              this.importer.deleteRelationship({
+                id: targetRelationship.id,
+                classFullName: targetRelationship.classFullName,
+              });
```

Comment:
same here, can we remove the above query now?

Reply:
changed